includes/openid-connect-generic-settings-page.php

```diff
@@ -296,6 +298,7 @@ private function get_settings_fields() {
 				'title'       => __( 'Enforce Privacy', 'daggerhart-openid-connect-generic' ),
 				'description' => __( 'Require users be logged in to see the site.', 'daggerhart-openid-connect-generic' ),
 				'type'        => 'checkbox',
+				'disabled'    => defined( 'OIDC_ENFORCE_PRIVACY' ),
```

Comment:
@ChloeTigre you didn't document in your PR, and didn't add this to the PHPStan bootstap, adding this additional constant. Please remove this or add the missing documentation and additional support.